WalletWasabi.Gui/Tabs/WalletManager/RecoverWalletView.xaml

```diff
@@ -29,6 +29,12 @@
               </DataTemplate>
             </ItemsControl.ItemTemplate>
           </ItemsControl>
+          <CheckBox IsChecked="{Binding ShowAdvancedOptions,Mode=TwoWay}"><TextBlock Text="Show advanced options"/></CheckBox>
+          <StackPanel Spacing="8" IsVisible="{Binding ShowAdvancedOptions}">
+            <controls:ExtendedTextBox Text="{Binding AccountKeyPath}" Watermark="Account Key Path" UseFloatingWatermark="True"/>
```

Comment:
I think we need a note here. We use the "Note that, " language syntax in the Send tab so it'd be consistent like this.

```suggestion
            <controls:ExtendedTextBox Text="{Binding AccountKeyPath}" Watermark="Account Key Path" UseFloatingWatermark="True"/>
            <TextBlock Text="Note that, Wasabi can only monitor bech32 addresses."/>
```